src/core/adapters/adaptverticalresults.js

```diff
@@ -0,0 +1,22 @@
+import Section from '../models/section';
+import adaptAppliedQueryFilter from './adaptappliedqueryfilter';
+import adaptResult from './adaptresults';
+
+/**
+ * Constructs an SDK Section model from an answers-core VerticalResult array
+ *
+ * @param {VerticalResults[]} verticalResults
+ * @param {Object<string, string>} urls keyed by vertical key
+ * @returns {@link Section}
+ */
+export default function adaptVerticalResults (verticalResults, urls) {
```

Comment:
To make it easier to use the "adaptVerticalResults" for both universal and vertical responses, what do you think about changing this to take in a single vertical result? I think the mental model could/should be that universal responses have a list of VerticalResults, so it would make sense that they'd need to call the adapt function on each one

Reply:
ya, that works!